_pages/kriskirov.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: kriskirov
+pageUrl: kriskirov.github.io
+timestamp: TODAYS-24-2017-04-Mo
+---
```

Comment:
@kriskirov Good question. Add the link that is show in your settings of the repository - as long as it's a valid URL and you omit the `http` portion, it should pass the test.